docs/spec.md

```diff
@@ -1884,46 +1884,30 @@ Produces an `output` tensor from a constant `value`.
 Performs an element-wise conversion from one element type to another on
 `operand` tensor and produces a `result` tensor.
 
-For conversions involving **integer-to-integer**, if there is an unsigned/signed
-overflow, the result is implementation-defined and one of the following:
-
-* mathematical result modulo $2^n$, where n is the bit width of the result,
-  for unsigned overflow. For signed integer overflow, wraps the result around
-  the representable range $[-2^{n-1},\ 2^{n-1} - 1]$.
-* saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow and
-  saturation to $2^n - 1$ (or $0$) for unsigned overflow.
```

Comment:
We used to specify possibilities for implementation-defined results, but stopped doing that in one of the pull requests in December. Let's apply this change to this formulation as well.